README.rst

```diff
@@ -32,6 +32,16 @@ Installation
 See doc/install.txt in the source code archive.
 Python 2.7.2 or later is needed.
 
+
+Install using Docker
+---------------------
+You can try linkchecker in an isolated enviroment without changing any local files or have to install all the dependencies via a Docker Container:
+
+``docker pull ellerbrock/linkchecker``
+
+https://hub.docker.com/r/ellerbrock/link-checker/
```

Comment:
i disagree. there are some official projects. here is the [debian repository](https://hub.docker.com/_/debian/). it says "official" on top. same as the [ubuntu](https://hub.docker.com/_/ubuntu/) one.

again, i don't really care if it's official or not. if it's unofficial, just say so in the patch. if it's official, we need to have access to it, have an automated procedure to rebuild it and so on.